plugins/rating/versions/validation/rating.js

```diff
@@ -22,6 +22,31 @@ module.exports = function createValidation (deps) {
 
   const schemas = {}
 
+  // ////////// //
+  // 2020-08-10 //
+  // ////////// //
+  schemas['2020-08-10'] = {}
+  schemas['2020-08-10'].getStats = () => ({
+    query: schemas['2019-05-20'].getStats.query
+      .fork('page', schema => schema.forbidden())
+      .keys({
+        // cursor pagination
+        startingAfter: Joi.string(),
+        endingBefore: Joi.string(),
+      })
```

Comment:
We should probably refactor this in a shared object (including `oxor` below), except if it’s too tricky with our use of Joi.

Reply:
Indeed, I'll make this change, thanks.